evadb/parser/create_statement.py

```diff
@@ -203,7 +201,44 @@ def __eq__(self, other):
         if not isinstance(other, CreateDatabaseStatement):
             return False
         return (
-            self.database_name == other.database_name
+            self.name == other.name
+            and self.if_not_exists == other.if_not_exists
+            and self.engine == other.engine
+            and self.param_dict == other.param_dict
+        )
+
+    def __hash__(self) -> int:
+        return hash(
+            (
+                super().__hash__(),
+                self.name,
+                self.if_not_exists,
+                self.engine,
+                hash(frozenset(self.param_dict.items())),
+            )
+        )
+
+    def __str__(self) -> str:
+        return (
+            f"CREATE DATABASE {self.name} \n"
+            f"WITH ENGINE '{self.engine}' , \n"
+            f"PARAMETERS = {self.param_dict};"
+        )
+
+
+class CreateApplicationStatement(AbstractStatement):
```

Comment:
We can let one class inherit the other to avoid duplicate code.

Reply:
Sure